.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
The script should be split to two parts in first place: `setup_example_app.sh` and `clean/teardown_example_app.sh`. Then the test can just call subprocess in `setup_module`.